wordpress-mobile/WordPress-iOS
# Remove `unseenPostCount` feature flag.

Fixes #15483 

This removes the `unseenPostCount` feature flag.

To test:
- Go to the Reader.
- Select `Filter` on a filterable tab.
- Verify the unseen count is still displayed.

PR submission checklist:
